Automattic/wp-calypso
# FSE: Add a site title block.

#### Changes proposed in this Pull Request

A basic site title block for use with Full Site Editing

#### Testing instructions

- Open up your FSE environment.
- Add a Site Title block in a FSE template.
- Notice that the Site Title block reflects the site's title.
- Update the Site Title and 'force dirty' the state by editing something else in the post contents.
- Click "Publish..."
- Refresh the page and notice that the site title has been updated.

Fixes #33106
